lib/models/room.js

```diff
@@ -750,3 +778,16 @@ module.exports = Room;
  *   if (newTags["favourite"]) showStar(room);
  * });
  */
+
+/**
+ * Fires whenever a room's account_data is updated.
+ * @event module:client~MatrixClient#"Room.accountData"
+ * @param {event} event The account_data event
+ * @param {Room} room The room whose account_data was updated.
+ * @example
+ * matrixClient.on("Room.accountData", function(event, room){
+ *   if (event.getType() === "m.room.colorscheme") {
```

Comment:
Unless this is part of the spec, can we not use `m.*` please. `com.example.colorscheme` would be a better example.